Detections/SecurityEvent/UserAccountAddedToPrivlegeGroup_1h.yaml

```diff
@@ -64,23 +64,31 @@ query: |
   | extend UserPrincipalName = tostring(EventData.UserPrincipalName)
   | extend SubjectUserName = tostring(EventData.SubjectUserName)
   | project TimeGenerated, EventID, Computer, SimpleMemberName, MemberName, MemberSid, TargetUserName, TargetDomainName, TargetSid, UserPrincipalName, SubjectUserName, SubjectUserSid
+  | extend HostName = tostring(split(Computer, ".")[0]), DomainIndex = toint(indexof(Computer, '.'))
+  | extend HostNameDomain = iff(DomainIndex != -1, substring(Computer, DomainIndex + 1), Computer)
+  | extend AccountName = tostring(split(SimpleMemberName, "@")[0]), AccountUPNSuffix = tostring(split(SimpleMemberName, "@")[1])
```

Comment:
This may not work since MemberName is not always populated.  Let me have a look at how we should improve this.  Go ahead and remove from the PR for now.